server/src/formulas/weightedAverage.ts

```diff
@@ -38,9 +39,29 @@ async function calculateWeightedAverage(
   };
 }
 
+const codeSnippet: string =
+`const weightedAverage = (nums, weights) => {
+  const [sum, weightSum] = weights.reduce(
+    (acc, w, i) => {
+      acc[0] = acc[0] + nums[i] * w;
+      acc[1] = acc[1] + w;
+      return acc;
+    },
+    [0, 0]
+  );
+  return sum / weightSum;
+};`;
+
+const name: string = 'Weighted average';
+
+const attributes: Array<string> = ['maxPoints', 'minRequiredPoints', 'weight'];
```

Comment:
`maxPoints` and `minRequiredPoints` should be removed, again related to #303. But this can also be done in another PR.

Reply:
removed